<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QB-IPL-2331" failures="1" tests="3" name="com.tests.LoginTest" time="64.680" errors="0" timestamp="2022-04-08T09:47:44 IST" skipped="1">
  <testcase classname="com.tests.LoginTest" name="verifyForgotPasswordButtonTest" time="14.676">
    <failure message="expected [Forgot your passwords?] but found [Forgot your password?]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Forgot your passwords?] but found [Forgot your password?]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.tests.LoginTest.verifyForgotPasswordButtonTest(LoginTest.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:262)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:829)
]]>
    </failure>
  </testcase> <!-- verifyForgotPasswordButtonTest -->
  <system-out/>
  <testcase classname="com.tests.LoginTest" name="verifyForgotPasswordButtonTest" time="22.887">
    <skipped/>
  </testcase> <!-- verifyForgotPasswordButtonTest -->
  <system-out/>
  <testcase classname="com.tests.LoginTest" name="verifyLoginTest" time="27.117"/>
  <system-out/>
</testsuite> <!-- com.tests.LoginTest -->
